posthog/warehouse/models/table.py

```diff
@@ -42,10 +55,10 @@
     "UInt16": IntegerDatabaseField,
     "UInt32": IntegerDatabaseField,
     "UInt64": IntegerDatabaseField,
-    "Float8": IntegerDatabaseField,
-    "Float16": IntegerDatabaseField,
-    "Float32": IntegerDatabaseField,
-    "Float64": IntegerDatabaseField,
+    "Float8": FloatDatabaseField,
+    "Float16": FloatDatabaseField,
+    "Float32": FloatDatabaseField,
+    "Float64": FloatDatabaseField,
```

Comment:
I managed to mess this up prior 😬 All sources will resync and this will get corrected, but, manually linked files won't - we may want a temporal workflow that just re-validates manually linked files schemas etc